CityOfDetroit/bloom
# Update detroitDefault image

# Pull Request Template

## Issue


## Description

Replaced the default stock image with a revised image.  New image adjusts some of the colors, and removes the transparency that was in the prior PNG file.

<img width="912" alt="Screen Shot 2021-08-17 at 2 15 00 PM" src="https://user-images.githubusercontent.com/82653098/129779011-c252d96e-bd87-47b2-b1aa-bff51fbc6768.png">


## How Can This Be Tested/Reviewed?

View the listings page or any individual listing without a picture to see the updated image.
